index.js

```diff
@@ -94,5 +94,12 @@ function StringHelper () {
     return this
   }
 
+  this.when = function (expression, then, otherwise) {
+    let should = isFunc(expression) ? expression.call(this) : expression
+    if (should === true) this.cat(then)
```

Comment:
This is OK but you can always short circuit
```js
if (should === true) return this.cat(then)
return otherwise === undefined ? this : this.cat(otherwise)
```